plugins/linux/ifplugin/linuxcalls/netlink_api.go

```diff
@@ -51,27 +52,27 @@ type NetlinkAPIWrite interface {
 	SetInterfaceMTU(ifName string, mtu int) error
 	// RenameInterface changes interface host name
 	RenameInterface(ifName string, newName string) error
+	// InterfaceExists verifies interface existence
+	InterfaceExists(ifName string) (bool, error)
```

Comment:
This should be in read as well. All calls that do not modify anything should be in read.